Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I don't use makefile in my other projects, and I'm not really confident with it.